python/qpid_dispatch_internal/tools/command.py

```diff
@@ -83,6 +83,11 @@ def connection_options(options, title="Connection Options"):
                      help="Trusted Certificate Authority Database file (PEM Format)")
     group.add_option("--ssl-password", action="store", type="string", metavar="PASSWORD",
                      help="Certificate password, will be prompted if not specifed.")
+    group.add_option("--no-verify-host-name", action="store_true", default=False,
```

Comment:
I'd suggest: --verify-peer-name type=bool, default=true. Bool options with negative names are confusing and "host" is a bit vague - peer is more clearly the _other_ host. Also the help text is very long, maybe "Verify the peer host name matches the certificate. Default true, setting to false is insecure ."